### ISSUE TITLE
Add an option to allow overwrite uploads

### ISSUE THREAD

**Issue by ntsk** [ntsk/fastlane-plugin-amazon_appstore#2]
If the upload is performed while the next version has already been created, it will fail.
If you want to update the contents of the next version before release, you need to delete the next version from the console UI and re-run it, which is tedious.
(For example, if you have a prior release on Google Play and know that a bug will occur.)

For this reason, I would like to add an option to make a DELETE request first and then POST.

```
[07:01:00]: Driving the lane 'android deploy_to_amazon' 🚀
[16:01:00]: ---------------------------------------
[16:01:00]: --- Step: upload_to_amazon_appstore ---
[16:01:00]: ---------------------------------------
[16:01:00]: Fetching access token...
[16:01:00]: Creating new edits...
[16:01:01]: {:httpCode=>412, :message=>"Precondition Failed", :errors=>[{:errorCode=>"error_new_version_creation_not_allowed", :errorMessage=>"Cannot create a new 'edit' for the app in it's current state."}]}
```

**Comment by tmm1** [comment]
Would be great. Are you still working on / using this plugin?

**Comment by ntsk** [comment]
@tmm1 
Yes, I am still using this plugin and will update it when necessary.
If this feature is needed, I am open to adding it.

**Comment by tmm1** [comment]
Cheers. I appreciate you cleaning up the code and switching to faraday etc. I switched to this plugin and it's working nicely.
